requirements/system/user-client.tex

```diff
@@ -1,9 +1,49 @@
 \subsection{User Client}
-\begin{enumerate}
- \item UCs must be accessible through a CLI.
- \item UCs must provide to the user the same functionality as if the user was working directly on a work machine.
- \item UCs must allow the user to specify an operating system on which the job should be run.
- \item UCs must allow the user to specify minimum hardware requirements for a job: RAM, CPU thread count, GPU memory/model/speed.
- \item UCs must allow the user to specify a priority for a job.
-\end{enumerate}
+
+
+  \subsubsection{Functional Requirements} 
+  \begin{itemize}
+    \item [FRUC01] Logging in to the central server
+    \item [FRUC02] Logging out from the central server
+    \item [FRUC05] Queueing new jobs
+    \item [FRUC06] Querying the state of jobs
+    \item [FRUC07] Canceling jobs
+    \item [FRUC08] Receiving the output
+    \item [FRUC09] Setting job priorities 
+    \item [FRUC10] Checking the specifications of available containers
+    \item [FRUC11] Choosing the parameters of the container(preferred OS/environment)
+    \item [FRUC12] Choosing the running mode: "blocking mode"/ default mode
+    \item [FRUC13] Choosing the minimal hardware requirements for a job 
+    \item [FRUC14] Checking the state of the entire server
+  \end{itemize}
+
+  \subsubsection{Non Functional Requirements} 
+  \begin{itemize}
+    \item [NFRUC01] Responses from the server must take less 10ms
+    \item [NFRUC02] Users can't queue more than 5 jobs at the same time 
+  \end{itemize}
+
+  \subsubsection{Optional Requirements} 
+  \begin{itemize}
+    \item [ORUC01] Saving a private image for future usage
+    \item [ORUC02] Scheduling a job to run at a specific time
+    \item [OR03] Choosing a preferred container to use
```

Comment:
What do you mean by preferred container?
There should be no ambiguity as to which container the worker client will run as a job.